### PR TITLE
Release notes for 6.5.0 & updated README.

### PR DESCRIPTION
**Pull request by mwz**
Did I miss anything? Is there anything you would like to add from your side @BalmungSan?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ sonar-scala
 
 **SonarQube plugin for static code analysis of Scala projects.**
 
-Intended for [SonarQube 6.7+ LTS](https://www.sonarqube.org/downloads) and Scala 2.11/2.12.
+Intended for [SonarQube 6.7+ LTS](https://www.sonarqube.org/downloads) and Scala 2.11/2.12.  
+It does not support Windows.
```
Comment (BalmungSan) [inline README.md]: Uhm... maybe we should be a little bit more precise with this?

Something like:
> "Running analysis from a windows machine is not supported"

Also, I'm not sure if all the plugin fail, or only Scoverage?

File: README.md [hunk 2]
```diff
@@ -91,10 +101,11 @@ export SONAR_SCANNER_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=y
 Once you've done that, `sonnar-scanner` should display the following message `Listening for transport dt_socket at address: 8000`. You can now attach your IDE to the process on port `8000`, set breakpoints and debug the code.
 
 # Credits
-This project is a continuation of sonar-scala plugin, which was originally developed by [Sagacify](https://github.com/Sagacify/sonar-scala) and integrates code from two other SonarQube plugins [Sonar Scalastyle Plugin](https://github.com/NCR-CoDE/sonar-scalastyle) and [Sonar Scoverage Plugin](https://github.com/RadoBuransky/sonar-scoverage-plugin).
+This project is a continuation of sonar-scala plugin, which was originally developed by [Sagacify](https://github.com/Sagacify/sonar-scala) and integrates code from [Sonar Scalastyle Plugin](https://github.com/NCR-CoDE/sonar-scalastyle).
 
 Many other projects have been used as an inspiration, here is a list of the main ones:
```
Comment (BalmungSan) [inline README.md]: IMHO, we should add the old scapegoat plugin here too.

File: build.sbt [hunk 3]
```diff
@@ -60,7 +60,7 @@ packageOptions in (Compile, packageBin) += Package.ManifestAttributes(
   PluginManifest.ISSUE_TRACKER_URL -> "https://github.com/mwz/sonar-scala/issues",
   PluginManifest.ORGANIZATION -> "Michael Wizner",
   PluginManifest.ORGANIZATION_URL -> "https://github.com/mwz",
-  PluginManifest.DEVELOPERS -> "Augustin Borsu, Michael Wizner",
+  PluginManifest.DEVELOPERS -> "Michael Wizner, Luis Miguel Mejía Suárez",
```
Comment (BalmungSan) [inline build.sbt]: Hey Thanks 😄

File: build.sbt [hunk 4]
```diff
@@ -119,4 +119,4 @@ releaseProcess := Seq[ReleaseStep](
 // Test
 parallelExecution in Test := false
 logBuffered in Test := false
-testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-oDF")
+testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-oDTF")
```
Comment (BalmungSan) [inline build.sbt]: I'm not familiar with scalatest arguments, but I just give quick read of [this](http://www.scalatest.org/user_guide/using_the_runner), and I think you're missing some values after these arguments.
Unless what I have seen has nothing to do with this, in which case just ignore it

Comment (mwz) [reply build.sbt]: So the `T` flag which I added is to show an extra reminder of failed and cancelled tests at the end. I find it quite useful as I don't need to scroll through all of the tests to see the failures - I'll add a comment.

File: README.md (hunk 1 above)
Comment (mwz) [reply README.md]: I updated this in 204ae17.

> Also, I'm not sure if all the plugin fail, or only Scoverage?

I'm not sure either - it's worth investigating this as a part of #65.

File: README.md (hunk 2 above)
Comment (mwz) [reply README.md]: done

File: build.sbt (hunk 3 above)
Comment (mwz) [reply build.sbt]: Well deserved!

File: build.sbt (hunk 4 above)
Comment (BalmungSan) [reply build.sbt]: Oh Nice, I'll use these in my projects too 😄

**Comment by BalmungSan** [comment]
All good now :+1:

**Pull request merged**